lib/nexmo_developer/app/presenters/improve_page_presenter.rb

```diff
@@ -7,13 +7,15 @@ def github_url
     @github_url ||= "https://github.com/#{docs_repo}/blob/#{ENV.fetch('branch', 'master')}/#{path_to_url}"
   end
 
+  def docs_repo
+    @docs_repo ||= begin
+      path_to_url.include?('app/') ? 'nexmo/station' : YAML.safe_load(File.open("#{Rails.configuration.docs_base_path}/config/business_info.yml"))['docs_repo']
+    end
+  end
+
   private
 
   def path_to_url
     @document_path&.gsub("#{Rails.configuration.docs_base_path}/", '')
```

Comment:
@bencgreenberg unfortunately, this doesn't quite fix the issue, we need to update the `path_to_url`, it should include `lib/nexmo_developer` in the case the file lives in station

Reply:
What's a better way to determine if it's in Station or not?